monitoring/tools/check_dead_queues.py

```diff
@@ -15,18 +15,20 @@ def check_deadletter_queues(
 ):
     bad_queue_messages = []
     dead_queues = get_queues(contains="deadletter")
+    environment = None
     for dead_queue in dead_queues:
         queue_size = int(dead_queue.attributes.get("ApproximateNumberOfMessages", 0))
         if queue_size > 0:
             queue_name = dead_queue.url.split("/")[-1]
+            if environment is None:
```

Comment:
When would the environment not be None?

Reply:
I initiate environment as None, so the first time in the for it'll get environment and after it will skip.